### DISCUSSION TITLE
Add stopwatch

### DISCUSSION THREAD

**Discussion by hitanshu-mehta** [cpeditor/cpeditor#1004] [Ideas]
Hi!
I usually keep track of the time taken to solve the problem. I think a stopwatch would be a useful feature.
I have added this feature to my fork. I would like to open a PR if it is needed. Please let me know what do you think? :) 

<details>
<summary>It looks something like this.</summary>

![Screenshot_20211104_223101](https://user-images.githubusercontent.com/44025541/140385607-a51543c6-4058-49f1-a461-9c49a9d38082.png)
</details>

**Comment by coder3101** [comment]
Yes it would certainly be a great addition. However, I think it should be inside a setting that user can enable or does it makes sense to have it always visible? What do you think @ouuan ?

**Reply by ouuan** [reply]
It should be a setting

**Reply by hitanshu-mehta** [reply]
Preferences -> Appearance -> general seems a good place for this setting?

**Reply by ouuan** [reply]
> Preferences -> Appearance -> general seems a good place for this setting?

It seems that this setting doesn't suit any section very well :thinking: I'd prefer Actions->Stopwatch

**Comment by hitanshu-mehta** [comment]
Great! I will create an issue and start working on PR.

**Comment by coder3101** [comment]
Extensions -> Stopwatch or Timer

**Reply by ouuan** [reply]
Extensions require external tools. So the stopwatch is not an extension.
